src/js/web-components/video-player/VideoPlayer.js

```diff
@@ -145,6 +145,30 @@ export default class extends HTMLElement {
             metadata.images = [videoThumbnail];
             mediaInfo.metadata = metadata;
 
+            const subtitles = this.internal.videoData['video-subtitles'] || [];
```

Comment:
How about merging both?
```js
const defaultSubtitles = this.internal.videoData["video-subtitles"].find(
  (subtitle) => subtitle.default
);
```

Reply:
@beaufortfrancois Can't be done currently – the first two videos have no subtitles, meaning that with those `this.internal.videoData["video-subtitles"] === undefined`.